src/wallet/rpcwallet.cpp

```diff
@@ -4132,7 +4141,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "encryptwallet",                    &encryptwallet,                 {"passphrase"} },
     { "wallet",             "getaddressesbylabel",              &getaddressesbylabel,           {"label"} },
     { "wallet",             "getaddressinfo",                   &getaddressinfo,                {"address"} },
-    { "wallet",             "getbalance",                       &getbalance,                    {"dummy","minconf","include_watchonly"} },
```

Comment:
Dummy should stay otherwise it breaks calls using named parameters.

Reply:
"dummy" has NEVER worked as a named parameter. The only calls using named parameters where this ever worked, used the name "account"

Reply:
Just tested v0.17.0 and the following works:
```
bitcoin-cli -regtest --named getbalance account="*"
5097.99999647
bitcoin-cli -regtest --named getbalance dummy="*"
5097.99999647
```